css/index.css

```diff
@@ -46,25 +49,36 @@ h1 {
   line-height: 1.4;
   
   */
-  
+  .content-row p{
+    font-size: 16px;
+    font-family: roboto;
+    padding: 10px;
+    line-height: 1.4;  
+  }
   
   /*======= Create a selector for ALL images in the site to have the following styles: 
   
     border-radius: 10px;
   
   */
+  .content-row img{
+    border-radius: 10px;
+  }
```

Comment:
Create a selector for all images. You are correctly selecting images, but not ALL of them.
This is a selector for all images nested within .content-row

All images would be:
img {
  border-radius: 10px;
}